tests/test_aird_filters.py

```diff
@@ -2,18 +2,129 @@
 # SPDX-License-Identifier: Apache-2.0
 
 """Tests for aird-filters applied after rendering a diagram"""
+import pytest
+
 import capellambse
+from capellambse import aird
+from capellambse import model as model_
 
 COMP_PORT_FILTER_DIAG = "[LAB] Test Component Port Filter"
+EX_ITEMS_FILTER_DIAG = "[SDFB] Test ExchangeItem Filter"
+DEFAULT_ACTIVATED_FILTERS = frozenset(
+    {
+        "ModelExtensionFilter",
+        "hide.overlappedfunctional.chains.label.filter",
+        "hide.overlappedfunctional.chains.icon.filter",
+    }
+)
+EX_ITEMS_FILTER = "show.exchange.items.filter"
+NAME_AND_EX_ITEMS_FILTER = "show.functional.exchanges.exchange.items.filter"
+
+
+def test_diagram_has_activated_filters(
+    model_5_2: capellambse.MelodyModel,
+) -> None:
+    diag: model_.diagram.Diagram = model_5_2.diagrams.by_name(
+        EX_ITEMS_FILTER_DIAG
+    )
+
+    assert diag.filters == DEFAULT_ACTIVATED_FILTERS
+
+
+def test_add_activated_filter_to_diagram(
+    model_5_2: capellambse.MelodyModel,
+) -> None:
+    diag: model_.diagram.Diagram = model_5_2.diagrams.by_name(
+        EX_ITEMS_FILTER_DIAG
+    )
+
+    diag.filters.add(EX_ITEMS_FILTER)
+
+    assert EX_ITEMS_FILTER in diag.filters
+
+
+@pytest.mark.parametrize("filter_name", DEFAULT_ACTIVATED_FILTERS)
+def test_remove_activated_filter_on_diagram(
+    model_5_2: capellambse.MelodyModel, filter_name: str
+) -> None:
+    diag: model_.diagram.Diagram = model_5_2.diagrams.by_name(
+        EX_ITEMS_FILTER_DIAG
+    )
+
+    diag.filters.remove(filter_name)
+
+    assert diag.filters == DEFAULT_ACTIVATED_FILTERS - {filter_name}
+
+
+def test_remove_activated_filter_fails_if_not_active(
+    model_5_2: capellambse.MelodyModel,
+) -> None:
+    diag: model_.diagram.Diagram = model_5_2.diagrams.by_name(
+        EX_ITEMS_FILTER_DIAG
+    )
+
+    with pytest.raises(KeyError):
+        diag.filters.remove(EX_ITEMS_FILTER)
 
 
 def test_component_ports_filter_is_applied(
     model: capellambse.MelodyModel,
 ) -> None:
-    diag = model.diagrams.by_name(COMP_PORT_FILTER_DIAG)
-    diag = diag.render(None)
+    diag: model_.diagram.Diagram = model.diagrams.by_name(
+        COMP_PORT_FILTER_DIAG
+    )
+
+    diagram = diag.render(None)
+
     assert all(
         d.hidden
-        for d in diag
+        for d in diagram
         if d.port and d.styleclass and d.styleclass.startswith("CP_")
     )
+
+
+def test_fex_exchangeitems_filter_is_applied(
+    model_5_2: capellambse.MelodyModel,
+) -> None:
+    diag: model_.diagram.Diagram = model_5_2.diagrams.by_name(
+        EX_ITEMS_FILTER_DIAG
+    )
+
+    diag.filters.add(EX_ITEMS_FILTER)
+    diagram = diag.render(None, sorted_exchangedItems=False)
+    fex_edge = diagram["_yovTvM-ZEeytxdoVf3xHjA"]
```

Comment:
Considering the volatile nature of these auto-generated IDs, do you suppose it's a good idea to identify this object in a different way maybe? Perhaps make a simple diagram that only has this one single FEX displayed, and identify it by its type (edge) and styleclass (FunctionalExchange). That would prevent this test from breaking when Capella inevitably rewrites the IDs as part of normal usage.